KellyRyanGlobal/testing-app
# Kelly 4 create docker container deploy app

To test run the following

1. **Centos VM**: Open a terminal and run the following
2. `git pull kelly-4-create-docker-container-deploy-app`
3. Open `README.md`
4. Following the `Build Logger` steps in the README.md
5. Once the application builds verify the output looks like the following
```
[test.cc:22 test1] TRACE TRACE: 1
[test.cc:24 test1] TEST INFO: Hi!42
[test.cc:31 test2] NETWORK: this is network logging
[test.cc:78 main] HOST: hostname: 6dac7cfbfb5b
[test.cc:80 main] SUB INFO: info!
[test.cc:81 main] SUB ERROR: error!
```
